python/res/util/ui_return.py

```diff
@@ -44,6 +44,13 @@ def __init__(self , status):
             raise ValueError('Unable to construct UIReturn with status = %s' % str(status))
 
 
+    def __bool__(self):
+        if self.status() == UIReturnStatusEnum.UI_RETURN_OK:
+            return True
+        else:
+            return False
```

Comment:
please just return `self.status() == UIReturnStatusEnum.UI_RETURN_OK` and rewrite `__nonzero__` to `self.__bool__()` or something appropriate

Reply:
Yes, but you don't need two different implementations. :)

I was thinking something like
```python
def __bool__(self):
    return self.status() == UIReturnStatusEnum.UI_RETURN_OK

def __nonzero__(self):
    return self.__bool__()
```